python/ray/serve/_private/controller.py

```diff
@@ -877,6 +877,8 @@ def get_serve_instance_details(self) -> Dict:
         grpc_config = self.get_grpc_config()
         applications = {}
 
+        app_configs = self.get_app_configs() or {}
```

Comment:
> I'm seeing a test failure on test_get_applications_while_gcs_down

I believe it's because you're trying to access the KV store while the GCS is down. Could you change this to not fetch the app configs when there are no applications?

Reply:
Thanks! Just pushed this up 🤞🏻

Reply:
Looks like this is passing now! Re-requested your review